marlowe-runtime-web/server/Language/Marlowe/Runtime/Web/Server/REST/Contracts/Next.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# HLINT ignore "Use <&>" #-}
```

Comment:
Why?

Reply:
sometimes, when I'm implementing functions, hlint proposes reductions that not readable anymore.. but usually this is temporary, and in that case it doesn't apply anymore.. I'll remove it...